tests/helpers/core-manager.js

```diff
@@ -50,6 +50,20 @@ const destroyStream = (stream) =>
     stream.destroy()
   })
 
+/**
+ * @param {CoreManager} cmToAdd
+ * @param {CoreManager} cmToReceive
+ * @returns {Promise<void>}
+ */
+async function addWriterCores(cmToAdd, cmToReceive) {
```

Comment:
nit @EvanHahn the naming is slightly confusing, because this is actually adding the cores "owned" by one instance to another instance, but as "read" cores, not writer cores. But I'm not sure what a better naming would be.

Reply:
Renamed in #795.